draft-ietf-scitt-architecture.md

```diff
@@ -372,7 +372,8 @@ This section describes at a high level, the three main roles and associated proc
 ## Transparency Service
 
 Transparency Services MUST feature an Append-only Log.
-The Append-only Log is the verifiable data structure that registers Signed Statements and supports the production of Receipts.
+The Append-only Log is a verifiable data structure that accumulates all Signed Statements 
+registered by the Transparency Service and that supports the production of Receipts.
 
```

Comment:
```suggestion
Optionally, the Ledger may also record other collateral information for Auditors, such as governance information, records of management operations or security incidents, intermediate certificates used for Issuer authentication, etc.

```

Reply:
If it is just a different kind of payload, why must we be explicit with this?

Reply:
Various implementations will support many different scenarios. I'd suggest change to 375/376 should be reverted, and remove the additional 378

Reply:
The original wording here is much more preferable. The verifiabiliy depends on the ability to produce proofs (receipts) not where the signed statements are stored.

Reply:
This gives me something extra to think about, sounds like a generaly useful thing, but doesn't seem necessary architecture ?

Reply:
Resolving as not-needed, and implementation specific details.
@ad-l, please re-open with comments and changes if you feel we need to keep it